test/odbc/psqlodbc/sql_variant.cpp

```diff
@@ -0,0 +1,614 @@
+#include "psqlodbc_tests_common.h"
+
+const string BBF_TABLE_NAME = "master.dbo.sql_variant_table_odbc_test";
+// For BBF Connection
+//   Cannot prepend database name when creating/dropping view
+//   Must prepend database name when selecting from view
+const string BBF_VIEW_NAME = "dbo.sql_variant_view_odbc_test";
+const string PG_TABLE_NAME = "master_dbo.sql_variant_table_odbc_test";
+const string PG_VIEW_NAME = "master_dbo.sql_variant_view_odbc_test";
+const string COL1_NAME = "pk";
+const string COL2_NAME = "data";
+const string DATATYPE_NAME = "sys.sql_variant";
+const vector<pair<string, string>> TABLE_COLUMNS = {
+  {COL1_NAME, "INT PRIMARY KEY"},
+  {COL2_NAME, DATATYPE_NAME}
+};
+
+class PSQL_DataTypes_Sql_Variant : public testing::Test {
+  void SetUp() override {
+    if (!Drivers::DriverExists(ServerType::PSQL)) {
+      GTEST_SKIP() << "PSQL Driver not present: skipping all tests for this fixture.";
+    }
+    if (!Drivers::DriverExists(ServerType::MSSQL)) {
```

Comment:
Can we combine this? 
```
if (!Drivers::DriverExists(ServerType::PSQL)) || !Drivers::DriverExists(ServerType::MSSQL)) {...
```

Reply:
I'd prefer to have them separate. If we were only running this test, and we didn't include one of the driver, this would let us know which driver was not included.

Reply:
I agree with @ColinKYuen . Resolving this comment.